docs/user-guide/src/capm3/label_sync.md

```diff
@@ -0,0 +1,45 @@
+# Labels Synchronization between BaremetalHost and Kubernetes Nodes
+
+CAPM3 has mechanism to synchronize BaremetalHost labels with predefined
+prefixes to the corresponding Kubernetes node object on that
+BaremetalHost.
+
+## How to use?
+
+To use label synchronization user needs to define prefix(es) for labels. Only
+labels that fall within prefix set are synchronized. User defines prefixes with
+annotation in Metal3Cluster object by using
+**metal3.io/metal3-label-sync-prefixes** annotation key and gives prefixes as
+annotation value. Prefixes should be seperated by comma.
+
+In the following example we are defining two label prefixes for label
+syncronization: **test.foobar.io** and **my-prefix**.
```

Comment:
On L22 you say all prefixes must be DNS qualified, making `my-prefix` invalid example?